lib/custom_wizard/submission.rb

```diff
@@ -97,6 +97,21 @@ def self.get(wizard, user_id)
     new(wizard, data, user_id)
   end
 
+  def self.remove(submission)
```

Comment:
@fzngagan When you're passing an entire instance as the argument it's a sign that tthis should be an instance method, and the class method should be the alias method.

Reply:
@fzngagan I'm not sure this class method makes sense now. We should just have the instance method.